src/Text/Parser/TreeSitter/Language.hs

```diff
@@ -38,7 +38,7 @@ mkSymbolDatatype name language = do
 
   Module _ modName <- thisModule
   pure
-    [ DataD [] name [] Nothing (flip NormalC [] . mkName <$> namedSymbols) [ ConT ''Show, ConT ''Enum, ConT ''Eq, ConT ''Ord, ConT ''Bounded, ConT ''Ix ]
+    [ DataD [] name [] Nothing (flip NormalC [] . mkName <$> namedSymbols) [ DerivClause Nothing [ ConT ''Show, ConT ''Enum, ConT ''Eq, ConT ''Ord, ConT ''Bounded, ConT ''Ix ] ]
```

Comment:
This is needed because ghc 8.2 adds support for deriving _strategies_; e.g. some instances can be derived by `DeriveAnyClass`, while others might be derived via `GeneralizedNewtypeDeriving`. This can be expressed syntactically, and thus also needs to be expressible in the quasiquoted representation.